### PR TITLE
Add release calendar tile to homepage

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Add release calendar tile to homepage

### How to review

- Pull sixteens branch `feature/list-tiles` - https://github.com/ONSdigital/sixteens/pull/191
- Sanity check, browser/device test

### Who can review

Anyone

### CODE REVIEW

**Comment by rav-pradhan** [comment]
I've been able to test when there are no release calendar entries, and it looks fine 
image

Is there any way to test this with release calendar data locally while the controller is still wip?

File: assets/templates/homepage/latest-releases.tmpl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <h2 class="margin-top--0 margin-right--0 margin-bottom--0 margin-left--0">
+            <span class="tile__title">{{ localise "LatestReleases" .Language 1 }}</span>
+        </h2>
+    </header>
+    {{ if gt $releasesLen 0 }}
+    <ul class="tile__list list--neutral margin-bottom--0">
+        {{range $releases}}
+            <li class="tile__list-item">
+                <a href="{{ .URI }}" class="tile__link tile__list-item-title">{{ .Title }}</a>
+                <p class="tile__list-item-content margin-top--0 margin-bottom--0">{{ .ReleaseDste }}</p>
```
Comment (rav-pradhan) [inline assets/templates/homepage/latest-releases.tmpl:15]: `ReleaseDate` typo I'm assuming 🙈

Comment (jondewijones) [reply assets/templates/homepage/latest-releases.tmpl:15]: 🤦🤦🤦🤦🤦🤦

Comment (jondewijones) [reply assets/templates/homepage/latest-releases.tmpl:15]: 👍

File: assets/templates/homepage/latest-releases.tmpl [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{{ $releases := len .Data.ReleaseCalendar.Releases }}
```
Comment (dandwal) [inline assets/templates/homepage/latest-releases.tmpl:1]: This breaks as we are setting the releases to an integer but then wanting to iterate over it.

```suggestion
{{ $releases := .Data.ReleaseCalendar.Releases }}
```

Comment (jondewijones) [reply assets/templates/homepage/latest-releases.tmpl]: Good spot. C&P error I think

File: assets/templates/homepage/latest-releases.tmpl [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {{ if gt $releasesLen 0 }}
+    <ul class="tile__list list--neutral margin-bottom--0">
+        {{range $releases}}
+            <li class="tile__list-item">
+                <a href="{{ .URI }}" class="tile__link tile__list-item-title">{{ .Title }}</a>
+                <p class="tile__list-item-content margin-top--0 margin-bottom--0">{{ .ReleaseDste }}</p>
+            </li>
+        {{ end }}
+    </ul>
+    <p class="tile__content margin-top--0 margin-bottom--0 font-size--18">{{ .Data.ReleaseCalendar.NumberOfReleases}} {{ localise "OtherReleasesThisWeek" .Language 1 }}</p>
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply assets/templates/homepage/latest-releases.tmpl:19]: Why doesn't the controller return the correct value? Less logic in the renderer then

**Pull request merged**